### PR TITLE
TRAC-891 Add utk_mods_etd_committee_member

### PR DESCRIPTION
**Pull request by cdeaneGit**
utk-fedora-fedoragsearch-solr-config(TRAC-891) was created from
utk-fedora-fedoragsearch-solr-config(instituional-repository)

The file slurp_all_MODS_to_solr.xslt was revised.
A template was made to take care of committee_members(s) and for each committee_member
creates a field utk_mods_etd_committee_member
Please test.
@CanOfBees @markpbaggett

### CODE REVIEW

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt [hunk 1]
```diff
@@ -64,6 +64,29 @@
     </field>
   </xsl:template>
 
+
+  <!-- the following template creates an _ms field for committee members -->
+  <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Committee member') or 
+    (mods:role/mods:roleTerm='committee member')]" mode="utk_ir_MODS">
+
+    <xsl:for-each>
```
Comment (CanOfBees) [inline fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt]: Same comments as on PR #10 for TRAC-886; this stylsheet won't compile: `xsl:for-each` has a **mandatory** select attribute. 

This won't quite work the way the template is articulated. Take a look at the [`xsl:for-each` for mods:abstract(s)](https://github.com/utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config/blob/institutional_repository/fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt#L85-L92).

**Review by CanOfBees — CHANGES_REQUESTED** [review]

**Comment by cdeaneGit** [comment]
Today revised slurp_all_MODS_to_solr.xslt and pushed to utk-fedora-fedoragsearch-solr-config(TRAC-891).
Ingested one file as userA.  All tests were successful.  Details in JIRA TRAC-691.
This code is ready to be merged.
@markpbaggett @CanOfBees

**Review by CanOfBees — APPROVED** [review]

**Comment by CanOfBees** [comment]
@cdeaneGit these changes look good. @markpbaggett can you review when you have a moment?

**Comment by markpbaggett** [comment]
@CanOfBees and @cdeaneGit , in the future, if we have a review done and it looks good, feel free to switch assignee and merge.  That being said, this has merge conflicts that need to be sorted.  This really should be dealt with by the person who opened the p.r. -- not the reviewer or merger. @cdeaneGit, can  you look at this and figure out why your file conflicts with the existing code?

**Comment by cdeaneGit** [comment]
I revised the template for utk_mods_etd_name_author_ms per Bridger's comment regarding conflicts with the institutional_repository branch.  I added fode for the field utk_mods_etd_author_orcid_s.  I pushed to
utk-fedoragsearch-solr-config(TRAC-891) branch.
I hope this is ready for merge.
@markpbaggett @CanOfBees

**Comment by markpbaggett** [comment]
@cdeaneGit this still says "This branch has conflicts that must be resolved." This means that something is out of whack between these 2 branches.  You can resolve through the browser or from the CLI.

**Comment by cdeaneGit** [comment]
This branch has no conflicts with the base branch
Ready for merge
@CanOfBees @markpbaggett

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt [hunk 2]
```diff
@@ -429,6 +426,17 @@
         </xsl:attribute>
         <xsl:value-of select="$value"/>
       </field>
+    </xsl:if>
```
Comment (CanOfBees) [inline fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt]: @cdeaneGit hey, I'm confused as to where/why this change came in. Am I missing something somewhere? Any suggestions or reminders would be most helpful. Thanks!

Edit: referring to lines 429-439

**Review by CanOfBees — CHANGES_REQUESTED** [review]

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt (hunk 2 above)
Comment (cdeaneGit) [reply fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt]: The way I resolved the conflicts was to pull all the code that was in institutional_repository/slurp_all_MODS_to_solr.xslt into TRAC-891/slurp_all_MODS_to_solr.xslt.  The comparison between the two branches kept having changes that were outside of the template I was working on.  So where this particular change came from, I do not know.

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt [hunk 3]
```diff
@@ -73,6 +115,13 @@
     </field>
   </xsl:template>
 
+  <!-- the following template adds a utk_mods_ir_publication field -->
```
Comment (markpbaggett) [inline fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt:118]: @cdeaneGit, this is confusing.  You're adding code to your files that is already in institutional_repository. I'm pretty sure that means if we merge we'll have 2 of the same template.

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt [hunk 4]
```diff
@@ -37,32 +37,74 @@
   </xsl:template>
 
   <!-- utk_ir_MODS mode -->
-  <!-- the following template creates an _ms field for author(s) -->
-  <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Author') or (mods:role/mods:roleTerm='author')]" mode="utk_ir_MODS">
+
```
Comment (markpbaggett) [inline fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt:40]: I'm confused.  Why are there all these new lines about author?  This should be about  committee member.

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt [hunk 5]
```diff
@@ -37,32 +37,74 @@
...
+
+  <!-- JIRA TRAC-875 Define utk_mods_etd_name_author_ms in Solr -->
```
Comment (markpbaggett) [inline fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt:42]: More author stuff.

**Review by markpbaggett — CHANGES_REQUESTED** [review]
@cdeaneGit, apologies, but I'm going to defer to @CanOfBees on this.  This p.r. is doing a lot of stuff and it's making me anxious.  The p.r. says it addresses TRAC-891 but it touches all this other stuff as well.  I personally don't feel comfortable merging, and at this point, honestly think closing and reopening is safest.

Perhaps @CanOfBees disagrees so I'm going to defer to him for his thoughts.

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt (hunk 3 above)
Comment (cdeaneGit) [reply fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt:118]: Looking at this again.

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt (hunk 5 above)
Comment (cdeaneGit) [reply fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt:42]: I think at this point, the simplest thing for me to do is to start over.  I have the template code for the committee-member.  I can close this pull request, make a new branch, add my template code and  nothing else and resubmit a pull request.  Where should I start the new branch from?
institutional_repository or master?
@markpbaggett @CanOfBees

**Comment by cdeaneGit** [comment]
I think at this point, the simplest thing for me to do is to start over.  I have the template code for the committee-member.  I can close this pull request, make a new branch, add my template code and  nothing else and resubmit a pull request.  Where should I start the new branch from?
institutional_repository or master?
@markpbaggett @CanOfBees
